src/pages/WorldSchedule/WorldSchedule.tsx

```diff
@@ -0,0 +1,193 @@
+import { useMemo } from "react";
+import { useForm } from "react-hook-form";
+import { useAsyncFn } from "react-use";
+import { faArrowLeft } from "@fortawesome/free-solid-svg-icons";
+
+import { ADMIN_IA_WORLD_PARAM_URL } from "settings";
+
+import { updateWorldScheduleSettings } from "api/world";
+
+import {
+  convertDateFromUtcSeconds,
+  convertUtcSecondsFromInputDateAndTime,
+} from "utils/time";
+import { generateUrl } from "utils/url";
+
+import { worldScheduleSchema } from "forms/worldScheduleSchema";
+
+import { useWorldBySlug } from "hooks/worlds/useWorldBySlug";
+import { useWorldParams } from "hooks/worlds/useWorldParams";
+
+import { AdminPanel } from "components/organisms/AdminVenueView/components/AdminPanel";
+import { AdminShowcase } from "components/organisms/AdminVenueView/components/AdminShowcase";
+import { AdminSidebar } from "components/organisms/AdminVenueView/components/AdminSidebar";
+import { EventsView } from "components/organisms/AdminVenueView/components/EventsView";
+
+import { AdminDateTime } from "components/molecules/AdminDateTime";
+import { AdminSection } from "components/molecules/AdminSection";
+import { AdminTitle } from "components/molecules/AdminTitle";
+import { AdminTitleBar } from "components/molecules/AdminTitleBar";
+import { FormErrors } from "components/molecules/FormErrors";
+import { LoadingPage } from "components/molecules/LoadingPage";
+import { SubmitError } from "components/molecules/SubmitError";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+import { NotFound } from "components/atoms/NotFound";
+
+import "./WorldSchedule.scss";
+
+const HANDLED_ERRORS: string[] = [
+  "start",
+  "startDate",
+  "startTime",
+  "end",
+  "endDate",
+  "endTime",
+];
+
+export interface WorldScheduleFormInput {
+  startDate: string;
+  startTime: string;
+  endDate: string;
+  endTime: string;
+}
+
+export const WorldSchedule = () => {
+  const { worldSlug } = useWorldParams();
+  const { isLoaded, world } = useWorldBySlug(worldSlug);
+
+  const defaultValues = useMemo<WorldScheduleFormInput>(() => {
+    const {
+      inputFormattedDateSegment: startDate,
+      inputFormattedTimeSegment: startTime,
+    } = convertDateFromUtcSeconds(world?.startTimeUnix ?? NaN);
```

Comment:
Interesting approach `?? NaN`.

Could be useful to have a comment explaining how `null` or `undefined` just don't work but `NaN` does.

Reply:
It was copy pasted from the regular schedule, good thing that you pointed it out. The reason it because `NaN` is actually a number. But maybe we should just return `0` instead.